src/fastmcp/client/tasks.py

```diff
@@ -33,8 +42,9 @@ class TasksGetResult(BaseModel):
     status: Literal[
         "submitted", "working", "completed", "failed", "cancelled", "unknown"
     ]
-    keepAlive: int | None = None
-    pollFrequency: int | None = None
+    createdAt: str
+    ttl: int | None = None
+    pollInterval: int | None = None
```

Comment:
Ah okay, we should move all of these temporary low-level SDK pydantic shims for different requests/responses into that new `_temporary_sep_1686_shims.py` file we're going to create, these shouldn't live here